sunpy/map/tests/test_mapbase.py

```diff
@@ -1026,7 +1021,6 @@ def test_superpixel_fractional_inputs(generic_map):
 @given(pc=matrix_meta('pc'))
 def test_resample_rotated_map_pc(pc, method):
     smap = make_simple_map()
```

Comment:
Change this to use the fixture

Reply:
I think the reason is:

```
Function-scoped fixtures are not reset between examples generated by
`@given(...)`, which is often surprising and can cause subtle test bugs.
```
But I can work around this.